doc/nrf/integrations.rst

```diff
@@ -1,10 +1,11 @@
-.. _ecosystem_integration:
+.. _integrations:
 
-Ecosystems integration
-######################
+Integrations
+############
 
-The following user guides describe the integration of various ecosystems and tools with |NCS|.
-The |NCS| is also integrated with the following ecosystems that are documented in private repositories:
+An |NCS| Integration allows for using a third-party product from within the |NCS|.
```

Comment:
```suggestion
An |NCS| Integration allows for using third-party and Nordic products within the |NCS|.
```
Since, nRF cloud is a Nordic product, may be we could change it as "third-party and Nordic products"? Thoughts?
If so, I can make the changes in the Integration template also.

Reply:
I am ok with that.